state/interface.go

```diff
@@ -18,6 +18,16 @@ const (
 	PeerAccountsState AccountsDbIdentifier = 1
 )
 
+// TriePruningIdentifier is the type for trie pruning identifiers
+type TriePruningIdentifier byte
```

Comment:
maybe move this to a `data.go` or `triePruningIdentifier.go` in the same package? for modified hashes you created a dedicated file

Reply:
done, moved to types.go